components/aws/sagemaker/tests/integration_tests/conftest.py

```diff
@@ -148,6 +149,9 @@ def kfp_client():
     kfp_installed_namespace = utils.get_kfp_namespace()
     return kfp.Client(namespace=kfp_installed_namespace)
 
+@pytest.fixture(scope="session")
+def k8s_client():
```

Comment:
its better to create a new session everytime see https://github.com/aws-controllers-k8s/test-infra/blob/main/src/acktest/k8s/resource.py#L119-L122

optional: update k8s version https://github.com/kubeflow/pipelines/blob/master/components/aws/sagemaker/tests/integration_tests/environment.yml#L16. Should we update all dependencies to latest in here and switch python version to 3.8?

Reply:
The old components rely on older boto3/sagemaker versions so its probably not possible.

Reply:
These are just test dependencies, not directly related to source code